wireapp/wire-android-reloaded
# feat: minimize ongoing call part2

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

[AR-1348](https://wearezeta.atlassian.net/browse/AR-1348)

### Issues

Couldn't minimize ongoing call

### Causes (Optional)

No action was taken on collapse button

### Solutions

Add navigate back on collapse button and component view to return to ongoing call if there is any for only two screens:
- ConversationScreen
- HomeScreen

### Dependencies (Optional)

Needs releases with:

- [X] #847 

### Testing

Tests were done manually.

#### How to Test

- Open App
- receive/start a call
- click minimize button in ongoing call screen
- status bar should be green with `RETURN TO CALL`
- Click `RETURN TO CALL` should open the ongoing call.

### Notes (Optional)

There are two possible issues:
- System back button doesn't properly collapse the ongoing call screen
- Microphone wrong state when re-opening ongoing call screen

both will be handled in a separate ticket.

### Attachments (Optional)
<img src="https://user-images.githubusercontent.com/5890660/184100119-eb9c0f88-d9c4-4a43-8f64-a267ff98110f.png" width="200" height="400" alt="Conversation Screen with RETURN TO CALL in Status Bar" />
